t/http-router.t

```diff
@@ -1923,4 +1923,60 @@ throws-like { bad-request }, X::Cro::HTTP::Router::OnlyInHandler, what => 'bad-r
     }
 }
 
+{
+    my $app = route {
+        get -> 'index.html' {
+            static-resource 'index.html';
+        }
+        get -> 'test.txt' {
+            static-resource 'folder/test.txt';
+        }
+        get -> 'folder-indexes' {
+            static-resource 'folder', :indexes(['test.txt']);
+        }
+        get -> 'root-indexes1' {
+            static-resource '', :indexes(['index.html']);
+        }
+        get -> 'root-indexes2' {
+            static-resource :indexes(['index.html']);
+        }
```

Comment:
I miss a test that covers the case where we pass multiple arguments to `static-resource`, e.g. `static-resource 'folder', 'text.txt'`. Given folks will probably use it for things like `get -> 'css', *@path { static-resource 'css', @path }` or similar, we should cover that.